translations/invoice-renderer.ro.xlf

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file source-language="en" target-language="ro" datatype="plaintext" original="file.ext">
+        <header>
+            <tool tool-id="symfony" tool-name="Symfony"/>
+        </header>
+        <body>
+            <trans-unit id=".Bji8X5" resname="label.invoice_renderer">
+                <source>label.invoice_renderer</source>
+                <target>Document factură</target>
+            </trans-unit>
+            <trans-unit id="N6juwc4" resname="default">
+                <source>default</source>
+                <target>Factură (implicit)</target>
+            </trans-unit>
+            <trans-unit id="lt2Y2LO" resname="timesheet">
+                <source>timesheet</source>
+                <target>Pontaj</target>
+            </trans-unit>
+            <trans-unit id="M82JFjR" resname="freelancer">
+                <source>freelancer</source>
+                <target>Freelancer</target>
```

Comment:
some translations are incomplete and missing (compare with latest version from in `en`)